free-3.md

```diff
@@ -10,41 +10,98 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+_Right now, without certain access and friends, it is difficult to play
+different types of card games competitively (Magic, Pokemon, Netrunner,
+etc.) because someone needs to buy the cards, which are not cheap, and
+practice playing, which requires someone else. Currently, most people
+choose not to play, for financial reasons or for lack of people to play
+with. Hopefully, this DSL would get them excited enough that they could 
+seek out people to play with because they would already know the game._
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+_These games have wide range of the types of cards that that can have. So,
+having a DSL allows the user to have variable rules and variable cards. It
```

Comment:
I think this is a really good point. This project is very interesting considering that certain card games, like Yu Gi Oh, has cards that are ridiculously powerful in the ways they can alter a game. You would need an expressive language to keep up with additional mechanics game designers release.